tests/integration/run-goth.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+echo REMOVING OLD GOTH ENV CONFIG
+rm /tmp/goth_interactive.env
+
+echo CREATING VENV
+rm -rf .envs/goth
+python -m venv .envs/goth
+source .envs/goth/bin/activate
+
+echo INSTALLING TOOLS
+.envs/goth/bin/python -m pip install --upgrade pip
+.envs/goth/bin/python -m pip install --upgrade setuptools wheel
+
+echo ISNTALLING DEPENDENCIES
+.envs/goth/bin/python -m pip install goth pytest pytest-asyncio pexpect
+
+echo CREATING ASSETS
+.envs/goth/bin/python -m goth create-assets .envs/goth/assets
+# disable use-proxy
+sed -Ezi 's/("\n.*use\-proxy:\s)(True)/\1False/mg' .envs/goth/assets/goth-config.yml
+
+echo STARTING NETWORK
+.envs/goth/bin/python -m goth start .envs/goth/assets/goth-config.yml &
+GOTH_PID=$!
+echo $GOTH_PID
+
+echo WAITING FOR NETWORK
```

Comment:
What are the actual conditions here?

Is it: if the file doesn't exist, wait 1 second, then if GOTH_PID doesn't exist, then abort?

Reply:
Wait for file (we make sure program will not continue without the file)
If the `GOTH_PID` doesn't exist anymore than there is no point waiting for the file as it will not be created

cat /tmp/goth_interactive.env will raise an error if something went wrong.